framework/resource/retryresource/resource.go

```diff
@@ -65,7 +65,7 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	}
 
```

Comment:
Note there is a PR hanging changing the retryresource: https://github.com/giantswarm/operatorkit/pull/153

I didn't merge yet, to make `dep update` in other projects still work. I'd like to merge when finishing all the Resource work. ETA is this week.

Reply:
I can chime in there.

Reply:
That's cool.

Reply:
Done.